dressForTheWeather/Assets.xcassets/Clothing Icons/hat.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Just a reminder to change the images to PDFs (for single scale)  and resize them to be the largest possible size (example maybe for these icons the max size is 100 X 100)

Reply:
I'll do this as a separate task in a future PR